### PR TITLE
Add bottom part of footer.

### PR DESCRIPTION
**Pull request by Irsanarisandy**
### DoD (Definition of Done) Checklist

- [x] There are no outstanding high priority defects or regressions
- [x] Testing notes and context added to the Jira card
- [x] Feature is tested in agreed devices and latest version of browsers**
- [x] The developer is confident that Acceptance Criteria have been met
- [x] No violations in aXe analysis ([Chrome Extension](https://chrome.google.com/webstore/detail/axe/lhdoppojpmngadmnindnejefpokejbdd))
- [x] Branch rebased against `develop`
- [x] Non-MVP enhancements have been captured in another card

##### * Design DOD
- [x] Accurately matches design unless there is a good reason not to
- [x] Hover states & interactions are correct
- [x] No regressions of other styles
- [x] Design checked by Dani

##### ** Agreed Browsers
- IE11 and latest versions of Edge, Chrome, Firefox

### Peer Review Checklist
- [x] Manually tested
- [x] Runs without issue locally
- [x] Browser/Device tested
- [x] Code is quality reviewed
- [x] Feedback is provided and documented
- [x] Matches all Acceptance Criteria

### CODE REVIEW

File: themes/kiwibuild/dist/js/app.js [hunk 1]
```diff
@@ -1 +1,124 @@
-!function(e){var t={};function n(r){if(t[r])return t[r].exports;var o=t[r]={i:r,l:!1,exports:{}};return e[r].call(o.exports,o,o.exports,n),o.l=!0,o.exports}n.m=e,n.c=t,n.d=function(e,t,r){n.o(e,t)||Object.defineProperty(e,t,{enumerable:!0,get:r})},n.r=function(e){"undefined"!=typeof Symbol&&Symbol.toStringTag&&Object.defineProperty(e,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(e,"__esModule",{value:!0})},n.t=function(e,t){if(1&t&&(e=n(e)),8&t)return e;if(4&t&&"object"==typeof e&&e&&e.__esModule)return e;var r=Object.create(null);if(n.r(r),Object.defineProperty(r,"default",{enumerable:!0,value:e}),2&t&&"string"!=typeof e)for(var o in e)n.d(r,o,function(t){return e[t]}.bind(null,o));return r},n.n=function(e){var t=e&&e.__esModule?function(){return e.default}:function(){return e};return n.d(t,"a",t),t},n.o=function(e,t){return Object.prototype.hasOwnProperty.call(e,t)},n.p="/",n(n.s=0)}([function(e,t,n){n(1),e.exports=n(2)},function(e,t){},function(e,t){}]);
\ No newline at end of file
+/******/ (function(modules) { // webpackBootstrap
```
Comment (pjayme) [inline themes/kiwibuild/dist/js/app.js]: this needs to be minified

File: themes/kiwibuild/src/scss/components/footer.scss [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  > .link {
+    color: $light-gray;
+    text-decoration: none;
+    text-transform: uppercase;
+
+    &:hover {
+      color: $light-gray;
+    }
+
+    > p {
```
Comment (pjayme) [inline themes/kiwibuild/src/scss/components/footer.scss]: I would avoid element selectors, always use classes

File: themes/kiwibuild/src/scss/components/footer.scss [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+      margin: 0;
+    }
+  }
+}
+
+.footer__break {
+  background-color: $footer-break;
```
Comment (pjayme) [inline themes/kiwibuild/src/scss/components/footer.scss]: not sure `footer-break` is an ideal color name, it doesn't really sound like a color, just name it closer to actual color name

File: themes/kiwibuild/templates/Includes/Footer.ss [hunk 4]
```diff
@@ -0,0 +1,35 @@
+<div class="footer">
+  <div class="footer__bottom">
+    <div class="container">
+      <div class="row">
+        <div class="col-sm-6 footer__kiwibuild">
+          <a class="$LinkingMode" href="$baseURL"/>
+            <img src="$themeDir/dist/images/kiwibuild-logo-white.svg" alt="Footer KiwiBuild Logo"/>
```
Comment (pjayme) [inline themes/kiwibuild/templates/Includes/Footer.ss]: nice practice here is to use brackets around `$` variables when mixed with standard strings 

i.e. `{$ThemeDir}/dist/images..`

File: themes/kiwibuild/templates/Includes/Footer.ss [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+          </a>
+        </div>
+        <div class="col-sm-6 footer__nzgovt">
+          <a class="$LinkingMode" href="https://www.govt.nz/"/>
+            <img src="$themeDir/dist/images/nz-govt-logo.svg" alt="Footer NZ Govt Logo"/>
```
Comment (pjayme) [inline themes/kiwibuild/templates/Includes/Footer.ss]: same as above

File: themes/kiwibuild/templates/Includes/Footer.ss [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+  <div class="footer__bottom">
+    <div class="container">
+      <div class="row">
+        <div class="col-sm-6 footer__kiwibuild">
+          <a class="$LinkingMode" href="$baseURL"/>
+            <img src="$themeDir/dist/images/kiwibuild-logo-white.svg" alt="Footer KiwiBuild Logo"/>
+          </a>
+        </div>
+        <div class="col-sm-6 footer__nzgovt">
+          <a class="$LinkingMode" href="https://www.govt.nz/"/>
```
Comment (pjayme) [inline themes/kiwibuild/templates/Includes/Footer.ss]: Does the `$LinkingMode` class get used here?

Comment (pjayme) [reply themes/kiwibuild/templates/Includes/Footer.ss]: I see, if it does get used this is a perfect case for using a modifier i.e. `footer__nzgovt--$LinkingMode`

File: themes/kiwibuild/templates/Includes/Footer.ss [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+          </a>
+        </div>
+      </div>
+      <hr class="footer__break">
+      <div class="row">
+        <div class="col-sm-8 col-md-9 col-lg-10 footer__menu">
+          <ul class="footer__legal-menu-list">
+            <% loop $MenuSet('BottomFooterMenu').MenuItems %>
+              <li class="footer__legal-menu-items">
+                <a class="$LinkingMode" href="$Link">
```
Comment (pjayme) [inline themes/kiwibuild/templates/Includes/Footer.ss]: Apply concept above

File: themes/kiwibuild/templates/Includes/Footer.ss (hunk 6 above)
Comment (Irsanarisandy) [reply themes/kiwibuild/templates/Includes/Footer.ss]: yup, i'm using link class

File: themes/kiwibuild/templates/Includes/Footer.ss [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+        </div>
+        <div class="col-sm-6 footer__nzgovt">
+          <a class="$LinkingMode" href="https://www.govt.nz/"/>
+            <img src="$themeDir/dist/images/nz-govt-logo.svg" alt="Footer NZ Govt Logo"/>
+          </a>
+        </div>
+      </div>
+      <hr class="footer__break">
+      <div class="row">
+        <div class="col-sm-8 col-md-9 col-lg-10 footer__menu">
```
Comment (pjayme) [inline themes/kiwibuild/templates/Includes/Footer.ss]: You can simplify this template a lot more if you simply make it `col-xs-12` and use `float: left | right` on the legal and copyright

File: themes/kiwibuild/templates/Includes/Footer.ss (hunk 7 above)
Comment (pjayme) [inline themes/kiwibuild/templates/Includes/Footer.ss]: When adding links in the CMS there is an option to open the link in a new window, don't forget to add this to your link :)

File: themes/kiwibuild/templates/Includes/Footer.ss [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+            <% loop $MenuSet('BottomFooterMenu').MenuItems %>
+              <li class="footer__legal-menu-items">
+                <a class="$LinkingMode" href="$Link">
+                  <p>$MenuTitle</p>
+                </a>
+              </li>
+            <% end_loop %>
+          </ul>
+        </div>
+        <div class="col-sm-4 col-md-3 col-lg-2 footer__copyright">
```
Comment (pjayme) [inline themes/kiwibuild/templates/Includes/Footer.ss]: Once you've applied the refactor above you can remove all these `col` classes and simply apply a `float: right` on the `footer__copyright`

File: themes/kiwibuild/templates/Includes/Footer.ss [hunk 10]
```diff
@@ -0,0 +1,35 @@
+<div class="footer">
+  <div class="footer__bottom">
+    <div class="container">
+      <div class="row">
+        <div class="col-sm-6 footer__kiwibuild">
```
Comment (pjayme) [inline themes/kiwibuild/templates/Includes/Footer.ss]: You can also simplify this too, just set it to `col-sm-12` then on a div below have `kiwibuild` - `float: left` and `nzgovt` - `float:right`

**Pull request merged**